mail-client/trojita/additional-files/trojita.rdef.in

```diff
@@ -0,0 +1,69 @@
+
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
```

Comment:
No idea, wich one the "Do not let the user start it 10 times in a row, because it makes nothing, but hey, it looks just bad in the Deskbar". You can freely push if you know the answer.